python/cuml/test/test_kmeans.py

```diff
@@ -72,8 +72,11 @@ def test_kmeans_sklearn_comparison(name, nrows):
         # since we are comparing 2 we just need to compare that both clusters
         # have approximately the same number of points.
         calculation = (np.sum(sk_y_pred) - np.sum(cu_y_pred))/len(sk_y_pred)
+        print(cuml_kmeans.score(X), kmeans.score(X))
```

Comment:
Was 2e-3 a coincidence here or was it the best tolerance you saw over a series of runs?

Reply:
No, it's what is used for 'noisy circles' calculation, so I just picked that number. While the score is accurate up to possibly 7th or 8th digit after the decimal point, I feel this difference is still a small enough number for if/when other tests are added. However, I could change this if you wish me to